### ISSUE TITLE
Implement TLS Capability for Secure Communication in Milvus CLI

### ISSUE THREAD

**Issue by jobbine-joseph** [zilliztech/milvus_cli#63]
The current iteration of the Milvus CLI lacks Transport Layer Security (TLS) capability, leaving communications unencrypted despite supporting authentication. This enhancement request aims to fortify the CLI by implementing TLS support for secure data transmission between the CLI and the server.

**Comment by gifi-siby** [comment]
Should we consider two-way authentication as well while adding TLS capability?

**Comment by gifi-siby** [comment]
The PR has been merged. Could you kindly share the timeline for the next release, as I’m eager to use the TLS capability?

**Comment by nameczz** [comment]
@gifi-siby Thanks for your pr. Will consider two-way auth later.
Already released yesterday. You can try v1.0.2.

**Comment by gifi-siby** [comment]
Yes, but I can see only source code in this release and other files

**Comment by gifi-siby** [comment]
Okay great. 
How can we change the documents regarding milvus_cli to mention TLS capability?

**Comment by nameczz** [comment]
Sure, feel free to create pr for documents.

**Comment by gifi-siby** [comment]
I can see only source code in this release and what about the other files?
